package.json

```diff
@@ -9,13 +9,9 @@
   "scripts": {
     "clean": "git clean -Xdf",
     "all": "turbo run build test format generate lint attw license-header",
-    "build": "turbo run build",
-    "test": "turbo run test",
-    "lint": "turbo run lint",
```

Comment:
Instead of root npm scripts running turbo to run npm scripts, we're moving to the same pattern as other repositories: There is only a single npm script "all" to run without turbo.